src/structured-data/structured-data-tags.js

```diff
@@ -140,6 +143,25 @@ function generateIsPartOfData(story = {}, publisherConfig = {}, pageType = "") {
   );
 }
 
+function generateIsPartOfDataForAmpLib(story = {}, publisherConfig = {}, pageType = "") {
```

Comment:
`generateIsPartOfData` function already takes `pageType`... can move this inside `generateIsPartOfData` instead of creating a new function IMO